server/helpers/responseHandler.js

```diff
@@ -1,10 +1,18 @@
 
 class Handler {
-  success = (code, message, payload, res) => res.status(code).json({
-    status: code,
-    message,
-    data: payload,
-  });
+  success = (code, message, payload, res) => {
+    if (payload != null) {
+      return res.status(code).json({
+        status: code,
+        message,
+        data: payload,
+      });
+    }
```

Comment:
@hakoemmy  make use of a statusHandler  helper file for "ALL" responses

Reply:
I am done with this